crates/proof-of-sql/src/base/database/arrow_array_to_column_conversion.rs

```diff
@@ -91,9 +92,11 @@ pub trait ArrayRefExt {
 impl ArrayRefExt for ArrayRef {
     #[cfg(any(test, feature = "test"))]
     #[cfg(feature = "blitzar")]
-    fn to_curve25519_scalars(
+    #[cfg(test)]
+    fn to_test_scalars(
```

Comment:
This is dead code. Let's just remove this and all the tests in this file that are testing it.

Reply:
Removed the code and its tests